ga4gh/datamodel/__init__.py

```diff
@@ -187,3 +218,21 @@ def _scanDataFiles(self, dataDir, patterns):
             atexit.register(_cleanupHtslibsMess, indexDir)
         if numDataFiles == 0:
             raise exceptions.EmptyDirException(dataDir, patterns)
+
+    def getFileHandle(self, dataFile):
+        if dataFile in self.memo_table:
```

Comment:
Just  a quick question about modifying the cache size during configuration:
during the tests, this configuration part doesn't seem to happen so we still need a default value for maxCacheSize when the module is loaded... do you have any suggestion?